view/frontend/templates/js/replacejs.phtml

```diff
@@ -24,6 +24,11 @@
  */
 if ($block->shouldDisableBoltCheckout()) return;
 
+//we need this script only for cart page, checkout page and other pages from white list
+//and for minicart is enabled
+//product page checkout has its own JS script
+if (!$block->isOnPageFromWhiteList() && !$block->isMinicartPage()) return;
```

Comment:
If Helper\Config::getMinicartSupport() is true (the default) this file will load on every page, including product pages. What happens in that case? Can we have both types of buttons at once, product checkout and the default checkout?

Reply:
Yes, it works

Reply:
Awesome!